CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ endif()
 
 set(IS_SUBDIRECTORY_INCLUDE true)
 find_package(aws-crt-cpp REQUIRED)
+find_package(aws-c-cal REQUIRED)
```

Comment:
This should be getting exported by aws-crt-cpp and not exposed outside of that. So if it’s not being picked up, we need to fix it there.